app/controllers/lists_controller.rb

```diff
@@ -2,7 +2,9 @@ class ListsController < ApplicationController
   before_action :authenticate_user!
 
   def show
+    @user = current_user
     @list = List.find(params[:id])
```

Comment:
Prefer user-scoped queries where possible to give added protection to private data. Line 6 would become:

```ruby
@list = current_user.lists.find(params[:id])
```